### PR TITLE
LPS-186301 Add Editable Action Panel

### PR DESCRIPTION
**Pull request by sandrodw3**

### CODE REVIEW

**Comment by sandrodw3** [comment]
ci:test:sf

**Comment by sandrodw3** [comment]
ci:test:sf

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/browser/components/page_structure/components/item_configuration_panels/EditableActionPanel.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+				editableValues: updateIn(
+					editableValues,
+					[
+						EDITABLE_FRAGMENT_ENTRY_PROCESSOR,
+						[item.editableId],
+						'config',
+						name,
+					],
+					() => value
+				),
```
Comment (victorg1991) [inline modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/browser/components/page_structure/components/item_configuration_panels/EditableActionPanel.js:123]: 😍

**Comment by victorg1991** [comment]
Just two minor comments, we can pass this pr and fix them aftewards though :)

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/common/components/MappingSelector.js [hunk 2]
```diff
@@ -64,6 +64,9 @@ function filterFields(fields, fieldType, filterLinkTypes) {
 			) {
 				return field.type === EDITABLE_TYPES.image;
 			}
+			else if (fieldType === EDITABLE_TYPES.action) {
+				return field.type === 'action';
```
[the remark this answers is not in the document]
Comment (sandrodw3) [reply modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/common/components/MappingSelector.js:68]: Yep! 😅

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/browser/components/page_structure/components/item_configuration_panels/EditableActionPanel.js (hunk 1 above)
Comment (sandrodw3) [reply modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/browser/components/page_structure/components/item_configuration_panels/EditableActionPanel.js:123]: 🥳

**Comment by sandrodw3** [comment]
ci:forward

**Comment by alessandromalves** [comment]
Resent here https://github.com/liferay-echo/liferay-portal/pull/12343

**Pull request closed without merging**